### ISSUE TITLE
Passing InputProps and InputLabelProps to SimpleFileUpload Component are not working.

### ISSUE THREAD

**Issue by chaitanya1019** [stackworx/formik-material-ui#73]
```
               <Field
                  component={SimpleFileUpload}
                  InputLabelProps={{
                    classes: {
                      root: classes.cssLabel,
                      focused: classes.cssFocused
                    }
                  }}
                  InputProps={{
                    classes: {
                      underline: classes.cssUnderline
                    }
                  }}
                  name="logoContent"
                  label="Upload College Logo*"
                  className={classes.logo}
                  required
                />
```

**Comment by JHenderson219** [comment]
@cliedeman I'll take a look at this and see about a fix.

**Comment by JHenderson219** [comment]
PR: #78 

I'm not as familiar with Typescript as I would like. If I didn't follow conventions that you would like to keep to, let me know and I can update the PR.

**Comment by cliedeman** [comment]
Released in v0.0.19
